circle.yml

```diff
@@ -2,7 +2,7 @@ general:
   branches:
     only:
       # - master # list of branches to build
-      - models
+      - devise
```

Comment:
why are you changing this file? once created and configure appropiately, that's it, no more changes